conf.d/fzf.fish

```diff
@@ -1,9 +1,10 @@
 # Set up the default, mnemonic key bindings unless the user has chosen to customize them
 if not set --query fzf_fish_custom_keybindings
+    set -l search_shell_variables '__fzf_search_shell_variables (set --show | psub) (set --names)'
```

Comment:
Can we also send the second argument into psub? Am worried about hitting argument limits. And also, then we'd be able to give an argument name to all the variables.

Reply:
> Am worried about hitting argument limits

![image](https://user-images.githubusercontent.com/44045911/103863011-083a8b80-50fb-11eb-91e3-7fac42cc7d85.png)

It's not like someone will have lots of variables.

Reply:
Ah good to know I don't need to worry about it. Sending it in as one file and then giving that file an argument name is self-documenting, however, so I still strongly prefer it.

Reply:
That's fine.